sdk/core/azure_core/CHANGELOG.md

```diff
@@ -1,16 +1,19 @@
 # Release History
 
-## 0.24.0 (Unreleased)
+## 0.24.0 (2025-05-02)
 
 ### Features Added
 
+- Added `TaskSpawner` abstraction to spawn asynchronous tasks for different async runtimes. Defaults to optional `tokio` runtime.
```

Comment:
Do we need to say optional, could it be "Defaults to `tokio` runtime."

Reply:
The tokio runtime is an optional feature. If the tokio feature is enabled, we will pick the tokio runtime over the native (thread based) runtime.

But since the tokio feature isn't on by default, we can't say that it defaults to the tokio runtime.

Reply:
That was exactly my reasoning. I didn't have "optional" in there originally and it felt too matter-of-factly.